src/test/metrics/stars-test.ts

```diff
@@ -0,0 +1,26 @@
+import test from 'ava';
+
+import {getStars} from '../../metrics/stars';
+import {startTestReplayServer} from '../replay-server';
+
+test.beforeEach(async (t) => {
+  const {server, client} = await startTestReplayServer(t);
+  t.context.server = server;
+  t.context.client = client;
+});
+
+test.afterEach.cb((t) => {
+  t.context.server.close(t.end);
+});
+
+test('WebComponents stars', async (t) => {
```

Comment:
I think you need to `git add` the test replay files.

Reply:
Oh sorry, just saw your comment. SG.

Reply:
Done